Makefile

```diff
@@ -494,9 +494,9 @@ pot: po
 	$(XGETTEXT_BIN) --from-code=utf-8 \
 		--keyword=C_:1c,2 --keyword=N_:1,2 --keyword=NC_:1c,2,3 \
 		--add-comments=@translators \
-		reader.lua `find frontend -iname "*.lua"` \
-		`find plugins -iname "*.lua"` \
-		`find tools -iname "*.lua"` \
+		reader.lua `find frontend -iname "*.lua" | sort` \
```

Comment:
@NiLuJe I figure this is probably the best approach, right? I noticed the whole freaking file seems to be rewritten all the time.

Reply:
Yup, sounds good :).